### PR TITLE
Add sum

### PR DESCRIPTION
**Pull request by nhanaa**

### CODE REVIEW

File: hello-spring/descriptors/ModuleDescriptor-template.json [hunk 1]
```diff
@@ -45,11 +45,11 @@
...
     "env": [
       { "name": "JAVA_OPTIONS", "value": "-XX:MaxRAMPercentage=66.0" },
-      { "name": "DB_HOST", "value": "postgres" },
+      { "name": "DB_HOST", "value": "localhost" },
       { "name": "DB_PORT", "value": "5432" },
-      { "name": "DB_USERNAME", "value": "folio_admin" },
-      { "name": "DB_PASSWORD", "value": "folio_admin" },
-      { "name": "DB_DATABASE", "value": "okapi_modules" },
+      { "name": "DB_USERNAME", "value": "postgres" },
+      { "name": "DB_PASSWORD", "value": "nhan3004" },
```
Comment (ncovercash) [inline hello-spring/descriptors/ModuleDescriptor-template.json]: i really hope you don't use this password for anything

File: hello-spring/src/main/resources/openapi/api.yaml [hunk 2]
```diff
@@ -51,3 +51,40 @@ paths:
                       - a
                       - b
                       - c
+  /sum/body:
+    get:
```
Comment (ncovercash) [inline hello-spring/src/main/resources/openapi/api.yaml]: oops, meant for this to be a POST.
```suggestion
    post:
```

File: hello-spring/src/main/java/org/folio/hello/SumController.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public final class SumController implements SumApi {
+
+  /** {@inheritDoc} */
+  @Override
+  public ResponseEntity<SumResponse> sumBodyGet(InputIntegers body) {
```
Comment (ncovercash) [inline hello-spring/src/main/java/org/folio/hello/SumController.java]: Meant for this to be a POST, oops

```suggestion
  public ResponseEntity<SumResponse> sumBodyPost(InputIntegers body) {
```

**Review by ncovercash — CHANGES_REQUESTED** [review]
Looks really good!  Quick fix that's my fault

File: hello-spring/descriptors/ModuleDescriptor-template.json (hunk 1 above)
Comment (nhanaa) [reply hello-spring/descriptors/ModuleDescriptor-template.json]: Oops, but nah that's just a quick password I used to set it up quickly.

Comment (ncovercash) [reply hello-spring/descriptors/ModuleDescriptor-template.json]: ok cool, just wanted to make sure! don't want you accidentally leaking something

**Review by ncovercash — APPROVED** [review]
LGMT, merge when you're ready!

**Pull request merged**